board/piksiv3/piksiv3_prod.dts

```diff
@@ -138,6 +138,25 @@
   status = "okay";
 };
 
+&i2c0 {
+  status = "okay";
```

Comment:
There are no slaves on this bus with kernel drivers.  In Duro the LEDs are connected here, so the we need kernel support for the I2C bus so it can be accessed from user space.